features/org.wso2.micro.integrator.server.feature/pom.xml

```diff
@@ -254,6 +254,7 @@
 <!--                                <bundleDef>org.wso2.carbon:org.wso2.carbon.addressing:${carbon.kernel.version}-->
 <!--                                </bundleDef>-->
                                 <!--Management API-->
+                                <bundleDef>org.wso2.orbit.com.nimbusds:nimbus-jose-jwt:${nimbus-jose.orbit.version}</bundleDef>
```

Comment:
add security related bundles to org.wso2.micro.integrator.security.server.feature

Reply:
Thanks. fixed